packages/web-app-files/src/router/deprecated.ts

```diff
@@ -54,8 +55,14 @@ export const buildRoutes = (): RouteConfig[] =>
       redirect: (to) =>
         createLocationSpaces('files-spaces-generic', {
           ...to,
-          params: { ...to.params, driveAliasAndItem: 'personal/home' }
-        })
+          params: {
+            ...to.params,
+            driveAliasAndItem: urlJoin('personal/home', to.params.item, { leadingSlash: false })
+          }
+        }),
+      meta: {
+        patchCleanPath: true
```

Comment:
Is this really needed? Or just a left over from trying?

Reply:
indeed not needed. removed it again.